dashboards-observability/public/components/visualizations/charts/bar/bar.tsx

```diff
@@ -10,80 +10,166 @@ import { LONG_CHART_COLOR, PLOTLY_COLOR } from '../../../../../common/constants/
 import { AvailabilityUnitType } from '../../../event_analytics/explorer/visualizations/config_panel/config_panes/config_controls/config_availability';
```

Comment:
Could we adjust the spaces in this file? I saw some statements are side to side, and some of them are having to empty lines. Please scan through to reformat.

Reply:
Thanks @mengweieric, We will scan through the files and do the required changes.